activities/membership-growth/membership-proposal-template.md

```diff
@@ -0,0 +1,116 @@
+
+We do this through codebase stewardship: supporting public organizations and developers (public and private) to develop and maintain code together.
+
+## Why the Foundation for Public Code exists
+
+Digital systems are increasingly core infrastructure for governments. Core government processes rely on digital systems to collect and manage their information, help inform decision making and execute their policy.
+
+As governments mature in their approach to their core digital systems, they face issues such as:
+
+* not having enough in-house technical talent for proactive architectural planning or maintenance of critical technology systems
+* defining what the private market can and cannot do reliably (e.g. store and manage critical data or lock-in situations)
+* understanding how to make their systems future-proof and ready for future innovation
+* avoiding 'technical debt' (e.g. building systems now that will cost a lot to maintain later)
+
+In light of this, it is important that public organizations approach their digital systems in a way that promotes effective and high quality delivery, and:
+
+* helps them work together with other public organizations - sharing software code and learnings - to reduce cost and make investments more effective
+* sets up a sustainable relationship with the market - helping different (types of) companies work together without a single point of failure (e.g. service designers with developers with system maintainers)
+* makes sure the public organizations are in control of their systems so they can respond to new citizen needs, policy goals or technologies
+
+## How the Foundation for Public Code can help your organization
+
+We are currently working with over 20 public organizations around Europe (including small municipalities, big cities, regional and national governments) on an open and collaborative process for developing reusable software components. Our aim is to help public organizations procure, develop and maintain code together.
+
+We developed a [Standard for Public Code](https://standard.publiccode.net/) for this purpose. The Standard is a collection of principles to help policy makers, managers, designers, developers and vendors work together to make code that is easy to understand, reuse and maintain.
+
+We also offer [codebase stewardship activities](https://about.publiccode.net/activities/codebase-stewardship/activities.html), which include a) reviewing the codebase (and all new additions to it) to make sure it complies with the Standard for Public Code, b) facilitating multi-stakeholder development between public and private organizations, c) helping with marketing to ensure public organizations understand the software and how to implement it, and d) supporting private sector parties to work effectively on these public-sector led software systems.
```

Comment:
```suggestion
We also do [codebase stewardship](https://about.publiccode.net/activities/codebase-stewardship/activities.html), which include a) reviewing the codebase (and all new additions to it) to make sure it complies with the Standard for Public Code, b) facilitating multi-stakeholder development between public and private organizations, c) helping with marketing to ensure public organizations understand the software and how to implement it, and d) supporting private sector parties to work effectively on these public-sector led software systems.
```

Reply:
changes 'we offer activities' to 'we do' + add 's' to include. Perhaps good to stay clear from being an activity-offering organization and the commercial mindset around that.